dubbo-dependencies-bom/pom.xml

```diff
@@ -108,6 +108,8 @@
         <cxf_version>3.1.15</cxf_version>
         <thrift_version>0.12.0</thrift_version>
         <hessian_version>4.0.38</hessian_version>
+        <protobuf-java>3.6.0</protobuf-java>
```

Comment:
simply use one property for both protobuf-java and protobuf-java-util, and name it 'protobuf-java_version'

Reply:
OK